src/components/navigation/Logo.tsx

```diff
@@ -1,26 +1,18 @@
-import { useTheme } from '@mui/material';
 import Box from '@mui/material/Box';
 import { useIntl } from 'react-intl';
-import logoDark from '../../images/dark/logo-estuary.png';
-import logoLight from '../../images/light/logo-estuary.png';
+import logo from 'images/pictorial-mark.png';
```

Comment:
Light mode for this application will likely be an assortment of teal. To achieve the glassmorphic effect, the background will need to be on the darker side of the medium tones. Therefore, it is very likely that the all-white versions of the logo will be used in both color modes. In the absence of a need for light and dark mode asset distinctions, the folders were removed.